conda-store-server/conda_store_server/app.py

```diff
@@ -87,7 +88,7 @@ class CondaStore(LoggingConfigurable):
     )
 
     store_directory = Unicode(
-        "conda-store-state",
+        str(Path.home() / ".conda-store" / "conda-store-state"),
```

Comment:
Add a variable for `Path.home() / ".conda-store"` and use it everywhere. For example: `CONDA_STORE_DIR`.

Reply:
The location of `CONDA_STORE_DIR` needs to be printed when the server is started.

Ideally: you can even print something like "Found existing conda-store.sqlite in `<dirname>`" or "Creating conda-store.sqlite in `<dirname>`". Same for the rest of top-level state files/dirs.

Otherwise, we'll run into problems when reproducing issues. People will have no idea that some state was created somewhere. Up until now, it hasn't been a problem because files are more visible in the current project dir, since it's tracked by git.